src/test/java/com/puppycrawl/tools/checkstyle/utils/CommonUtilTest.java

```diff
@@ -446,7 +447,29 @@ public void testGetUriByFilenameFindsAbsoluteResourceOnClasspath() throws Except
     }
 
     @Test
-    public void testGetUriByFilenameFindsRelativeResourceOnClasspath() throws Exception {
+    public void testGetUriByFilenameByClasspathPrefix() throws CheckstyleException {
+        final String filename =
+            "classpath:/" + getPackageLocation() + "/InputCommonUtilTest_empty_checks.xml";
```

Comment:
please update some (not all, as we need to be sure we keep compatibility) our configs to use this.
We have to use features we produce.